### PR TITLE
Sync ember-engines content with Ember Engines guides

### PR DESCRIPTION
**Pull request by villander**
cc: @jenweber @dgeb

### CODE REVIEW

**Comment by villander** [comment]
How can i fix this @jenweber ? 

image

**Comment by dgeb** [comment]
@villander it should be spelled `LinkedIn` (so just capitalize the "I")

**Comment by villander** [comment]
@dgeb `LinkedIn` don't work

File: guides/release/applications/ember-engines.md [hunk 1]
```diff
@@ -1,25 +1,23 @@
...
+Engines provide an antidote to these problems by allowing for distributed development, testing, and packaging of logically-grouped pieces of an application.
 
-If you are considering splitting up your application into engines just to reduce the amount of data that needs to be initially downloaded and increase the performance, Engines are not the right solution. Please check out the section on tree shaking and code splitting in projects like [Embroider](https://github.com/embroider-build/embroider).
+Engines are good for organizations that have multiple teams, where each team has their own area that is clearly separated from the others. The isolation is good when it matches organizational boundaries, but adds unnecessary complexity when there is not a good match.
 
+Engines are used by a number of organizations, including [LinkedIn](linkedin.com), [Chase](https://www.chase.com/), and [Square](https://squareup.com), to power sites with million of users.
```
Comment (dgeb) [inline guides/release/applications/ember-engines.md]:
```suggestion
Engines are used by a number of organizations, including [LinkedIn](https://www.linkedin.com/), [Chase](https://www.chase.com/), and [Square](https://squareup.com), to power sites with million of users.
```

**Comment by dgeb** [comment]
I am not sure if we need to add an ignore directive for the linter for this particular case.

File: guides/release/applications/ember-engines.md [hunk 2]
```diff
@@ -1,25 +1,23 @@
...
+Engines provide an antidote to these problems by allowing for distributed development, testing, and packaging of logically-grouped pieces of an application.
 
-If you are considering splitting up your application into engines just to reduce the amount of data that needs to be initially downloaded and increase the performance, Engines are not the right solution. Please check out the section on tree shaking and code splitting in projects like [Embroider](https://github.com/embroider-build/embroider).
+Engines are good for organizations that have multiple teams, where each team has their own area that is clearly separated from the others. The isolation is good when it matches organizational boundaries, but adds unnecessary complexity when there is not a good match.
 
+Engines are used by a number of organizations, including [LinkedIn](https://www.linkedin.com/), [Chase](https://www.chase.com/), and [Square](https://squareup.com), to power sites with million of users.
```
Comment (jenweber) [inline guides/release/applications/ember-engines.md]:
```suggestion
Engines are used by a number of large organizations to power sites with million of users.
```

**Pull request merged**